engines/sci/engine/kvideo.cpp

```diff
@@ -128,36 +128,35 @@ reg_t kShowMovie(EngineState *s, int argc, reg_t *argv) {
 	Common::ScopedPtr<Video::VideoDecoder> videoDecoder;
 
 	bool switchedGraphicsMode = false;
+	Graphics::PixelFormat screenFormat = g_system->getScreenFormat();
 
 	if (argv[0].isPointer()) {
 		Common::String filename = s->_segMan->getString(argv[0]);
 
 		if (g_sci->getPlatform() == Common::kPlatformMacintosh) {
 			// Mac QuickTime
 			// The only argument is the string for the video
+			videoDecoder.reset(new Video::QuickTimeDecoder());
 
-			// Switch to 16bpp graphics for Cinepak
```

Comment:
Why has this code been removed? The title of the PR says "Dither QuickTime videos in paletted screen modes", but with this change, we'll always use a paletted screen mode.

The whole point of this code is to actually switch to 16bpp, since the QT videos in question (the Mac KQ6 videos) are encoded in 16bpp. If we rip this code out, the screen will stay in paletted mode, which means that the video will ALWAYS be dithered. 
Why? What's the point of making this video look always dithered (i.e. ugly)?